templates/ui-frameworks/lit/collection/ui/src/{{dna_role_name}}/{{coordinator_zome_manifest.name}}/{{kebab_case collection_name}}.ts.hbs

```diff
@@ -51,29 +51,29 @@ export class {{pascal_case collection_name}} extends LitElement {
       this.signaledHashes = [{{#if (eq referenceable.hash_type "ActionHash")}}payload.action.hashed.hash{{else}}(payload.action.hashed.content as NewEntryAction).entry_hash{{/if}}, ...this.signaledHashes];
     });
   }
-  
+
   renderList(hashes: Array<{{referenceable.hash_type}}>) {
-    if (hashes.length === 0) return html`<span>No {{lower_case (plural referenceable.name)}} found{{#if (eq collection_type.type "ByAuthor")}} for this author{{/if}}.</span>`;
-    
-    return html`
+    if (!hashes.length) return html`<article>No {{lower_case (plural referenceable.name)}} found{{#if (eq collection_type.type "ByAuthor")}} for this author{{/if}}.</article>`;
 
-      <div style="display: flex; flex-direction: column">
-        ${hashes.map(hash => 
-          html`<{{kebab_case referenceable.name}}-detail .{{camel_case referenceable.name}}Hash=${hash} style="margin-bottom: 16px;" @{{kebab_case referenceable.name}}-deleted=${() => { this._fetch{{pascal_case (plural referenceable.name)}}.run(); this.signaledHashes = []; } }></{{kebab_case referenceable.name}}-detail>`
-        )}
+    return html`
+      <div>
+        ${hashes.map(hash => html`
+          <{{kebab_case referenceable.name}}-detail
+            .{{camel_case referenceable.name}}Hash=${hash}
+            @{{kebab_case referenceable.name}}-deleted=${() => { this._fetch{{pascal_case (plural referenceable.name)}}.run(); this.signaledHashes = []; } }
+          ></{{kebab_case referenceable.name}}-detail>
+        `)}
       </div>
     `;
   }
 
   render() {
     return this._fetch{{pascal_case (plural referenceable.name)}}.render({
-      pending: () => html`
-        <div style="display: flex; flex: 1; align-items: center; justify-content: center">
-          <mwc-circular-progress indeterminate></mwc-circular-progress>
-        </div>
-      `,
+      pending: () => html`<progress></progress>`,
       complete: (links) => this.renderList([...this.signaledHashes, ...links.map(l => l.target)]),
-      error: (e: any) => html`<span>Error fetching the {{lower_case (plural referenceable.name)}}: ${e.message}.</span>`
+      error: (e: any) => html`<article>Error fetching the {{lower_case (plural referenceable.name)}}: ${e.message}.</article>`
     });
   }
+
+  static styles = css`${unsafeCSS(sharedStyles)}`;
```

Comment:
I think in lit the more common way to do this would be to define the styles in a `.ts` file that exports the styles as a `css` template string instead of a `.css` file but I guess this would mess up with the reusability of the `index.css` file?

Reply:
Yep it would hinder reusability. I could still create one but I would still need to import it in every file and import the css file into it but I'd still end up using the `unsafeCSS` function to do that.

Reply:
With a `css` template string you'd not need to use `unsafeCSS`. But I see that it hinders code reusability across UI frameworks.
Another possible approach that would I think be a bit prettier from an optics standpoint would be to have a file `sharedStyles.ts` in the lit project (next to index.css) with the following content:
```
import { css, unsafeCSS } from "lit";
import styles from "./index.css";

export const sharedStyles = css`${unsafeCSS(styles)}`;
```
and then in the components where you need the styles you'd just import the styles from that file:
```
import { sharedStyles } from "./sharedStyles";

[...]

  static styles = sharedStyles;

````

Reply:
So you would only use the `unsafeCSS` function one single time when re-exporting them in the `sharedStyles.ts` file and it would not show up in all the components repeatedly.

Reply:
Good idea, I have also been contemplating another approach where I to create a class that extends `LitElement` i.e `BaseElement`, inside this class I apply the shared styles then for all other components in the app I extend the `BaseElement` rather than the `LitElement`

```ts
export class BaseElement extends LitElement {
  static styles = [
    unsafeCSS(css),
  ];
}

@customElement('holochain-app')
export class HolochainApp extends BaseElement {
  render() {
    return html`...`;
  }
}
```

What do you think?

Reply:
I think the method you suggest is much cleaner though, and doesn't involve changing too much code

Reply:
Your solution is interesting as well - I haven't thought of that. I don't know what's better...